packages/commonwealth/client/scripts/helpers/feature-flags.ts

```diff
@@ -3,4 +3,5 @@ export const featureFlags = {
   communityHomepage: process.env.FLAG_COMMUNITY_HOMEPAGE === 'true',
   sidebarToggle: process.env.FLAG_SIDEBAR_TOGGLE === 'true',
   newAdminOnboardingEnabled: process.env.FLAG_NEW_ADMIN_ONBOARDING === 'true',
+  communityStake: process.env.FLAG_COMMUNITY_STAKE === 'true',
```

Comment:
I add a placeholder for this ->  https://github.com/hicommonwealth/commonwealth/pull/6388#discussion_r1456583961. Will update once this PR is merged.

Reply:
thanks!